### PR TITLE
fixed #547

### LINKED ISSUE #547: Update hubCreateEntity task to use ES too
Right now hubCreateEntity just creates a directory. It should scaffold a barebones ES model def is requested.

### PR DESCRIPTION
**Pull request by paxtonhare**

### CODE REVIEW

File: marklogic-data-hub/src/main/java/com/marklogic/hub/deploy/commands/LoadUserModulesCommand.java [hunk 1]
```diff
@@ -188,7 +188,7 @@ else if (isEntityDir(dir, startPath.toAbsolutePath())) {
                                     InputStream inputStream = r.getInputStream();
                                     StringHandle handle = new StringHandle(IOUtils.toString(inputStream));
                                     inputStream.close();
-                                    entityDocMgr.write(URLEncoder.encode("/entities/" + r.getFilename(), "UTF-8"), meta, handle);
+                                    entityDocMgr.write("/entities/" + r.getFilename(), meta, handle);
```
Comment (dmcassel) [inline marklogic-data-hub/src/main/java/com/marklogic/hub/deploy/commands/LoadUserModulesCommand.java]: is this on purpose? doesn't seem related to the issue

Comment (paxtonhare) [reply marklogic-data-hub/src/main/java/com/marklogic/hub/deploy/commands/LoadUserModulesCommand.java]: I accidentally checked the URLEncode in the other day. this is me undoing it.

Comment (paxtonhare) [reply marklogic-data-hub/src/main/java/com/marklogic/hub/deploy/commands/LoadUserModulesCommand.java]: I should get this into develop for today's release and not on this branch.

**Comment by dmcassel** [comment]
Looks good to me
